{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -252,14 +299,104 @@ def env(name, default=NoDefaultValue, type_=str):
 }
 ACCOUNT_EMAIL_REQUIRED = True
 ACCOUNT_UNIQUE_EMAIL = False
-
+SOCIALACCOUNT_ADAPTER = (
+    '{{cookiecutter.project_slug}}.multisalesforce.adapter.CustomSocialAccountAdapter'
+)
 
 JS_REVERSE_JS_VAR_NAME = 'api_urls'
 JS_REVERSE_EXCLUDE_NAMESPACES = ['admin']
 
 
+# Redis configuration:
+
+REDIS_LOCATION = '{0}/{1}'.format(
+    env('REDIS_URL', default='redis://localhost:6379'),
+    0,
+)
+CACHES = {
+    'default': {
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': REDIS_LOCATION,
+        'OPTIONS': {
+            'CLIENT_CLASS': 'django_redis.client.DefaultClient',
+            'IGNORE_EXCEPTIONS': True,
+        },
+    },
+}
+RQ_QUEUES = {
+    'default': {
+        'USE_REDIS_CACHE': 'default',
+        'DEFAULT_TIMEOUT': 360,
+    },
+    'short': {
+        'USE_REDIS_CACHE': 'default',
+        'DEFAULT_TIMEOUT': 10,
+    },
+}
+
+
+# Logging
+
+LOG_REQUESTS = True
+LOG_REQUEST_ID_HEADER = "HTTP_X_REQUEST_ID"
+GENERATE_REQUEST_ID_IF_NOT_IN_HEADER = True
+REQUEST_ID_RESPONSE_HEADER = "X-Request-ID"
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': True,
+    'filters': {
+        'request_id': {
+            '()': 'log_request_id.filters.RequestIDFilter'
+        },
+        'job_id': {
+            '()': '{{cookiecutter.project_slug}}.logfmt.JobIDFilter'
+        }
+    },
+    'formatters': {
+        'verbose': {
+            '()': '{{cookiecutter.project_slug}}.logfmt.LogfmtFormatter',
+            'format': (
+                '%(levelname)s %(asctime)s %(module)s %(process)d '
+                '%(thread)d %(message)s'
+            ),
+        },
+    },
+    'handlers': {
+        'console': {
+            'level': 'DEBUG',
+            'class': 'logging.StreamHandler',
+            'filters': ['request_id'],
+            'formatter': 'verbose'
+        },
+        "rq_console": {
+            "level": "DEBUG",
+            "class": "rq.utils.ColorizingStreamHandler",
+            'filters': ["job_id"],
+            "formatter": "verbose",
+        },
+    },
+    'loggers': {
+        'django.db.backends': {
+            'level': 'ERROR',
+            'handlers': ['console'],
+            'propagate': False,
+        },
+        'django.server': {
+            'handlers': ['console'],
+            'level': 'INFO',
+            'propagate': False,
+        },
+        'rq.worker': {
+            "handlers": ["rq_console"],
+            "level": "DEBUG"
+        }
+    },
+}
```

Comment:
Something about this logging config gives me the jitters. I kinda want something more composable. I'll give it thought, and until I do, this is fine.